core/src/edu/rice/comp413fall2020grey/Common/metadata/ObjectMetadataField.java

```diff
@@ -0,0 +1,13 @@
+package edu.rice.comp413fall2020grey.Common.metadata;
+
+public abstract class ObjectMetadataField {
+    private final String fieldName;
+
+    public ObjectMetadataField(String fieldName) {
+        this.fieldName = fieldName;
+    }
+
+    public String getFieldName() {
+        return this.fieldName;
+    }
+}
```

Comment:
I'm a bit confused as to the point of this (not tot relevant as I am not privy to object location meetings/discussions), as this is essentially just a variable with no data. Like in the example metadata, these fields are simply just generic names, not names associated with the actual object instance

Reply:
old code

Reply:
ahh, sorry about that (also besides the few questions I had everything looks great!)